app/src/main/res/values-xhdpi/dimens.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <!-- Default screen margins, per the Android Design guidelines. -->
+  <dimen name="activity_horizontal_margin">16dp</dimen>
+  <dimen name="activity_vertical_margin">16dp</dimen>
+
+  <integer name="fade_in_duration_ms">1000</integer>
+  <integer name="fade_out_duration_ms">1000</integer>
+  <dimen name="audio_fragment_corner_radius">8dp</dimen>
+  <dimen name="audio_fragment_margin">28dp</dimen>
+  <dimen name="cellular_data_dialog_padding">24dp</dimen>
+  <dimen name="state_previous_next_button_radius">4dp</dimen>
+  <dimen name="train_checkbox_padding">18dp</dimen>
+  <dimen name="elevation_4dp">4dp</dimen>
+  <dimen name="elevation_0dp">0dp</dimen>
+  <dimen name="padding_72dp">72dp</dimen>
+  <dimen name="padding_24dp">24dp</dimen>
+  <dimen name="divider_margin_top">8dp</dimen>
+  <dimen name="divider_margin_bottom">8dp</dimen>
+  <dimen name="bottom_white_space">160dp</dimen>
+  <dimen name="padding_48">48dp</dimen>
+  <dimen name="padding_20">20dp</dimen>
+  <dimen name="margin_32">32dp</dimen>
+  <dimen name="margin_8">8dp</dimen>
```

Comment:
Remove these other dimens from this file. Just keep only those in which there is difference in value

Reply:
Done.